test/integration/targets/yum_repository/tasks/yum_repository_centos.yml

```diff
@@ -7,18 +7,20 @@
 - name: disable epel
   yum_repository:
     name: epel
-    description: EPEL yum repo
-    baseurl: https://download.fedoraproject.org/pub/epel/{{ ansible_distribution_major_version }}/{{ ansible_architecture }}/
     state: absent
 
 - name: disable epel (Idempotant)
   yum_repository:
     name: epel
-    description: EPEL yum repo
-    baseurl: https://download.fedoraproject.org/pub/epel/{{ ansible_distribution_major_version }}/{{ ansible_architecture }}/
     state: absent
   register: epel_remove
 
+- name: check return values
+  assert:
+    that:
+      - "epel_remove.repo == \"epel\""
```

Comment:
`"epel_remove.repo == 'epel'"` would make this (and other instances a little more readable